build.gradle.kts

```diff
@@ -40,7 +46,12 @@ promoteStagingRepo()
 
 subprojects {
     apply(plugin = "idea")
-    apply(plugin = "kotlin")
+    if (name.contains("android")) {
```

Comment:
This isn't super slick but results in minimal duplication. I'd love for the declaration of Android dependencies to be local to those projects, but then we can't add the common dependencies here. They could be pulled into a function somewhere but we'd have to call it from every applicable project.